src/main/java/dk/kb/ds/cumulus/export/CumulusExport.java

```diff
@@ -1,7 +1,79 @@
 package dk.kb.ds.cumulus.export;
 
+import dk.kb.cumulus.CumulusQuery;
+import dk.kb.cumulus.CumulusRecord;
+import dk.kb.cumulus.CumulusRecordCollection;
+import dk.kb.cumulus.CumulusServer;
+import dk.kb.cumulus.utils.ArgumentCheck;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+
 public class CumulusExport {
-    public static void main(String[] args) {
-        System.out.println("Cumulus exporter");
+    public static void main(String[] args) throws IOException, TransformerException, ParserConfigurationException {
+
+        try (CumulusServer server = new CumulusServer(Configuration.getCumulusConf())) {
+            String myCatalog = Configuration.getCumulusConf().getCatalogs().get(0);
+            CumulusQuery query = CumulusQuery.getQueryForAllInCatalog(myCatalog);
+            CumulusRecordCollection recordCollection = server.getItems(myCatalog, query);
+
+            File outputFile = new File("solrInputFile.xml");
+            OutputStream out = new FileOutputStream(outputFile);
+
+            ArgumentCheck.checkNotNull(out, "OutputStream out");
+            DocumentBuilderFactory docFactory = DocumentBuilderFactory.newInstance();
+            DocumentBuilder docBuilder = docFactory.newDocumentBuilder();
+            Document document = docBuilder.newDocument();
+            Element rootElement = document.createElement("add");
+            document.appendChild(rootElement);
+
+            for (CumulusRecord record : recordCollection) {
+                Element docElement = document.createElement("doc");
+                rootElement.appendChild(docElement);
+                try {
+                    String id = record.getFieldValueOrNull("guid");
+                    String titleC = record.getFieldValueOrNull("Titel");
+                    String dateC = record. getFieldValueForNonStringField("Date Time Digitized");
+                    String categoriesC = record.getFieldValueOrNull("Categories");
+                    String topicC = record.getFieldValueOrNull("Emneord");
+                    String copyrightC = record.getFieldValueOrNull("Copyright Notice");
+
+                    String[] attributeContent = {id, titleC, dateC, categoriesC, topicC, copyrightC};
+                    String[] attributeName = {"id", "title", "date", "categories", "topic", "copyright"};
```

Comment:
There is a preliminary list of Solr-fields at [ds-solr](https://github.com/Det-Kongelige-Bibliotek/ds-solr/blob/DIGISAM-115-start/template/conf/schema.xml). As I see it, the Cumulus `Emneord` is `topic` and `Categories` is `keyword`. But I am not 100% sure about that. `Date time digitized` is `created_date` (probably, maybe...).

Reply:
Resolved in the latest commit, thank you.